jujugui/static/gui/src/app/app.js

```diff
@@ -2727,8 +2721,7 @@ YUI.add('juju-gui', function(Y) {
       const setUpModel = model => {
         // Tell the model to use the new socket URL when reconnecting.
         model.set('socket_url', socketUrl);
-        // Store the existing credentials so that they can be possibly reused.
-        model.setCredentials(credentials);
+        this.user.controller = credentials;
```

Comment:
I'm not sure this is necessary any longer. I believe it's just going to set it to the same value it was at 2707